boscoseries/iReporter
# (chore) coverall: implement LFA feedback on coverall [finishes #162447712]

### Description of Task to be completed?
setup code coverage on coverall
### How should this be manually tested?
clone the repository
checkout into branch chore-server-setup-coveralls-162447712
- run npm test.
- code climate should show code quality and test coverage
### What are the relevant pivotal tracker stories?
#162447712
